engine/classes/Elgg/Database/UsersTable.php

```diff
@@ -549,34 +360,9 @@ public function setLastAction(ElggUser $user) {
 			return;
 		}
 
-		$query = "
-			UPDATE {$this->table}
-			SET
-				prev_last_action = last_action,
-				last_action = :last_action
-			WHERE guid = :guid
-		";
-
-		$params = [
-			':last_action' => $time,
-			':guid' => (int) $user->guid,
-		];
-
 		// these writes actually work, we just type hint read-only.
 		$user->prev_last_action = $user->last_action;
 		$user->last_action = $time;
```

Comment:
since this is an Entity attribute this still needs to be saved to the database

Reply:
done